src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -528,42 +544,130 @@ function Test-AnalysisServicesServerRestart
 	}
 }
 
+<#
+.SYNOPSIS
+Tests associate/dissociate gateway for server
+In order to run this test successfully, Following environment variables need to be set.
+A pre-configured onpremise gateway that is configured in the same location for testing default is East US 2.
+The testing user should have the permission to the gateway.
+ex. SET ASAZURE_TEST_ADMUSERS=asengsys@microsoft.com
+    SET ASAZURE_TESTUSER_PWD=somepassword
+	SET ASAZURE_TEST_ROLLOUT=stable.asazure.windows.net
+	SET ASAZURE_TEST_GATEWAY_NAME=azuresdktest
+#>
+function Test-AnalysisServicesServerGatewayAssociation
+{
+	try
+	{  
+		# Creating server
+		$location = "West US"
+		$resourceGroupName = Get-ResourceGroupName
+		$serverName = Get-AnalysisServicesServerName
+		New-AzureRmResourceGroup -Name $resourceGroupName -Location $location
+
+		# create a server with gateway
+		$serverCreated = New-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName -Location $location -Sku 'S0' -Administrator $env:ASAZURE_TEST_ADMUSERS
+		Assert-AreEqual $serverName $serverCreated.Name
+		Assert-AreEqual $location $serverCreated.Location
+		Assert-AreEqual "Microsoft.AnalysisServices/servers" $serverCreated.Type
+		Assert-AreEqual S0 $serverCreated.Sku.Name
+		Assert-True {$serverCreated.Id -like "*$resourceGroupName*"}
+		Assert-True {$serverCreated.ServerFullName -ne $null -and $serverCreated.ServerFullName.Contains("$serverName")}
+
+		start-sleep -s 1
```

Comment:
Use Wait-Seconds instead, so the sleep does not occur in playback mode: https://github.com/Azure/azure-powershell/blob/preview/src/Common/Commands.ScenarioTests.Common/Common.ps1#L374

Reply:
will do